udata_ckan/harvesters.py

```diff
@@ -134,8 +139,21 @@ def initialize(self):
         # status = self.get_status()
         # fix = status['ckan_version'] < '1.8'
```

Comment:
Maybe remove those?

Reply:
I hesitate, I wanted to have the minimal scope, but fine to me :+1: